cirq/linalg/decompositions_test.py

```diff
@@ -18,6 +18,7 @@
 import pytest
 
 import cirq
+from cirq.linalg.decompositions import kak_vector
```

Comment:
I didn't see how to run the test without this line. I tried adding `kak_vector` to some `__init__.py` files but that didn't work.

Reply:
You need to include `kak_vector` in the big import lists within `cirq/linalg/__init__.py` and `cirq/__init__.py`.

Reply:
Done.